<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="INBGQT-LT-0460" name="com.poc.twitter.test.POST_statuses_update_Test" tests="6" failures="1" timestamp="22 Jan 2019 15:04:24 GMT" time="0.040" errors="0">
  <testcase name="T04_Search_followers_count" time="0.006" classname="com.poc.twitter.test.POST_statuses_update_Test"/>
  <testcase name="T05_Search_friends_count" time="0.004" classname="com.poc.twitter.test.POST_statuses_update_Test"/>
  <testcase name="T01_Search_Source" time="0.010" classname="com.poc.twitter.test.POST_statuses_update_Test"/>
  <testcase name="T02_Search_Screen_name" time="0.007" classname="com.poc.twitter.test.POST_statuses_update_Test"/>
  <testcase name="T06_Search_Statuses_count" time="0.007" classname="com.poc.twitter.test.POST_statuses_update_Test">
    <failure type="java.lang.AssertionError" message="Status count is wrong! expected:&lt;181&gt; but was:&lt;190&gt;">
      <![CDATA[java.lang.AssertionError: Status count is wrong! expected:<181> but was:<190>
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.failNotEquals(Assert.java:834)
at org.junit.Assert.assertEquals(Assert.java:118)
at com.poc.twitter.test.POST_statuses_update_Test.T06_Search_Statuses_count(POST_statuses_update_Test.java:99)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- T06_Search_Statuses_count -->
  <testcase name="T03_Search_Name" time="0.006" classname="com.poc.twitter.test.POST_statuses_update_Test"/>
</testsuite> <!-- com.poc.twitter.test.POST_statuses_update_Test -->
